R/count_teleconnections.R

```diff
@@ -55,6 +55,101 @@ count_watershed_teleconnections <- function(data_dir,
   import_raster(paste0(data_dir, crop_file_path)) ->
     cropcover_USA
 
+      # START GCAM CROP CLASS BINDING
+
+      # Load in the GCAM classes CSV.
+      gcam_csv <- get_crop_mapping()
+
+      # Delete the rows that have crop types not present in CDL classes.
+      gcam_csv <- gcam_csv[-c(70, 100, 131, 45, 104, 182, 181),]
```

Comment:
Can this just be make to keep rows where `GCAM_commodity` is not NA?  I worry that row numbers may change over time when new data becomes available.